### ISSUE TITLE
Wont auto update in background

### ISSUE THREAD

**Issue by GodLikeInfernal** [Ilshidur/LoL-item-sets#8]
Minimize it and all it says is it thinks I have no connection or I have the wring folder selected which I don't, I even double checked the folder it's set to.

**Comment by Ilshidur** [comment]
Did you check if the website was up when you tried ? It happens that the website gets a lot of requests and sometimes has down times because of that.

Otherwise, it can be a permissions problem. Have you run the application with administrator rights ?

Other question : does this error only occurs when using the auto-update feature ?

**Comment by GodLikeInfernal** [comment]
It's only the auto update future but I didnt have admin rights turned on for it so ima try that and see if that fixes it.

**Comment by GodLikeInfernal** [comment]
Seems like setting it so it launch's with admin rights fixed it, thanks.

**Comment by GodLikeInfernal** [comment]
Uh no actually it still gave the same error even with admin rights.

**Comment by Ilshidur** [comment]
That seems to be a bug with the app.

Does it only happens when the application is minified in your task bar ? (after closing with the "Minimize on close" option checked ?)

**Comment by GodLikeInfernal** [comment]
Yes I have that checked, it doesn't even tell me I got a error until I open it back up from being minimized to my task bar.

**Comment by Ilshidur** [comment]
O.K., this sounds like a bug when the app can't update only when it's minified. Is that right ?

Anyway, thanks for the feedback. I will work on this as soon as I have some free time.

**Comment by GodLikeInfernal** [comment]
Alright, thank you.

**Comment by GodLikeInfernal** [comment]
and yeah only when its minified.
